spec/unit/honeybadger/rack/error_notifier_spec.rb

```diff
@@ -1,20 +1,21 @@
-require 'honeybadger/rack/error_notifier'
+require "honeybadger/rack/error_notifier"
 
-class BacktracedException < Exception
+class BacktracedException < RuntimeError
```

Comment:
I wonder if there was a reason we did this. The fact that it's named `BacktracedException`. 🤔